corehq/apps/couch_sql_migration/README.md

```diff
@@ -3,11 +3,73 @@
 The commands:
 
 ```sh
-./manage.py migrate_domain_from_couch_to_sql --help
+./manage.py couch_domain_report --help
 ./manage.py migrate_multiple_domains_from_couch_to_sql --help
+./manage.py migrate_domain_from_couch_to_sql --help
 ./manage.py couch_sql_diff --help
 ```
 
+## Setup
+
+First, from a machine where you have `commcare-cloud` installed, check if there
+are any Couch domains that need to be migrated:
+
+```sh
+commcare-cloud <env> django-manage couch_domain_report
+```
+
+You can stop here if the total at the bottom of the report is zero. Your
+environment does not need to be migrated. On the other hand, if the total is
+non-zero, proceed to setup and run migrations.
+
+The migration commands outlined in this document are typically run on a "limited
+release" on a machine in your CommCareHQ cluster, usually the `django_manage`
+machine. 8GB of RAM is recommended, although less may be necessary for small
+domains. It is also recommended to use a screen session so that network
```

Comment:
nit: minor reword + commcare-cloud has builtin support for tmux but not for screen

```suggestion
machine. 8GB of RAM is recommended, although small domains may require less.
It is also recommended to use a tmux or screen session so that network
```